tests/operators/test_arithmetic.py

```diff
@@ -14,6 +14,8 @@
 import numpy as np
 from hidet import ops
 
+hidet.option.cache_dir("temp")
```

Comment:
Hi Xin, can I know why we change the cache directory? 

FYI:
1. Hidet will store the operator cache under '<default cache dir>/test_cache' by default. See [here](https://github.com/hidet-org/hidet/blob/main/tests/conftest.py). 
2. We will clear the cache if any changes in our IR/passes. See [here](https://github.com/hidet-org/hidet/blob/67ddbeb5fddaf5f8339b0d2187f78076e30e4555/.github/workflows/tests.yaml#L61-L80).

Reply:
sorry this is an adhoc change that I forgot to revert